connexion/decorators/parameter.py

```diff
@@ -377,6 +376,9 @@ def _get_body_argument(
     if len(arguments) <= 0 and not has_kwargs:
         return {}
 
+    if operation.request_body == {}:
```

Comment:
Can you add a comment with some explanation here? It wasn't immediately clear to me that `operation.request_body == {}` was checking if a request body was defined in the specification.

Another option is that we add a boolean `reauest_body_defined` property to the `Operation` class which wraps this check, so it's more readable where it's used.